### PR TITLE
fix(ui): remove relative date filtering feature flag and related code [TCTC-6755]

### PR DESCRIPTION
**Pull request by Align-center**
## What
- Removing the `RELATIVE_DATE_FILTERING` feature flag that was only used in the `FIlterSimpleCondition` component and was preventing the display of the right operators when having a `dateTime` column selected. 
- Remove related code since this was the only place where the FF was used.

### CODE REVIEW

File: ui/tests/unit/filter-simple-condition-widget.spec.ts [hunk 1]
```diff
@@ -539,28 +440,8 @@ describe('Widget FilterSimpleCondition', () => {
...
-      createWrapper('until');
-      await wrapper.vm.$nextTick();
-      expect(wrapper.emitted().input).toHaveLength(1);
-      // should emit new value with 'le' operator replacing invalid 'until' operator
-      expect(wrapper.emitted().input[0]).toEqual([
-        { column: 'columnA', value: '', operator: 'le' },
-      ]);
-    });
-
     it('should transform an invalid "le" operator when relative date are enabled', async () => {
```
Comment (davinov) [inline ui/tests/unit/filter-simple-condition-widget.spec.ts:443]: Maybe we can remove the "when relative date are enabled" from test titles ?

**Review by davinov — APPROVED** [review]
Awesome !
Just a minor comment :

Thanks for cleaning this 👍👍👍

File: ui/tests/unit/filter-simple-condition-widget.spec.ts (hunk 1 above)
Comment (Align-center) [reply ui/tests/unit/filter-simple-condition-widget.spec.ts]: Yes you're right ! Fixed in [44bb611](https://github.com/ToucanToco/weaverbird/pull/1900/commits/44bb61154b74948aef85831742a9b376b2d83711) 🙏

**Review by alice-sevin — APPROVED** [review]
Topissime, merci beaucoup pour le cleaning :pray:

**Pull request merged**